src/components/PupilViews/PupilQuiz/PupilQuiz.view.tsx

```diff
@@ -121,6 +121,7 @@ const QuizInner = () => {
             tooltip={pickTooltip(currentQuestionData.answers)}
             isOpen={currentQuestionState?.mode === "incomplete"}
             tooltipPosition="top-right"
+            id="quiz-tooltip"
```

Comment:
This popped up on my branch too, unsure why Pupil squad didn't implement it aha

Reply:
yeah I assume they mustn't have updated components version yet 🤷‍♀️